scripts/migrate.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+#  ytcc - The YouTube channel checker
+#  Copyright (C) 2020  Wolfgang Popp
+#
+#  This file is part of ytcc.
+#
+#  ytcc is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ytcc is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with ytcc.  If not, see <http://www.gnu.org/licenses/>.
+
+import sqlite3
+import sys
+import textwrap
+from pathlib import Path
+
+
+def error(message: str):
+    print("Error:", message)
```

Comment:
I suggest sending errors to `stderr`
```suggestion
    print("Error:", message, file=sys.stderr)
```